### ISSUE TITLE
The "Most help wanted issues" option should not appear in the "sort types" dropdown menu.

### ISSUE THREAD

**Issue by alexandredev3** [max-programming/hacktoberfest-projects#178]
In my opinion, the **Most help wanted issues** option should not appear in the "sort types" dropdown menu. This is because users may want to sort repositories based on the most stars, most forks, or best match, while also being able to view the repository with the most help wanted issues.
Perhaps we could create a separate dropdown menu for the **Most help wanted issues** option? What do you guys think about this?

I would love to work on this!

**Comment by max-programming** [comment]
Yeah maybe it should be in a checkbox or something. You can work on this @alexandredev3

**Comment by THILAK6** [comment]
Hi @max-programming, Can I work on this please if it is still needed at this point of time?

**Comment by THILAK6** [comment]
> @THILAK6 Can you please wait for #212 to get merged? That is a breaking change

sure

**Comment by max-programming** [comment]
@THILAK6 Done
Now you can continue, make sure to get the latest code from `main`
